[CI] XpackWithMultipleProjectsClientYamlTestSuiteIT test {yaml=esql/10_basic/basic with documents_found} failing

**Build Scans:**
- [elasticsearch-periodic-platform-support #7751 / amazonlinux-2023_platform-support-aws](https://gradle-enterprise.elastic.co/s/xvpm7q4ec3xec)
- [elasticsearch-pull-request #68467 / part-2-windows](https://gradle-enterprise.elastic.co/s/eztdahxaekb3i)

**Reproduction Line:**
```
./gradlew ":x-pack:qa:multi-project:xpack-rest-tests-with-multiple-projects:yamlRestTest" --tests "org.elasticsearch.multiproject.test.XpackWithMultipleProjectsClientYamlTestSuiteIT.test {yaml=esql/10_basic/basic with documents_found}" -Dtests.seed=D676AAD22180945B -Dtests.locale=en-UM -Dtests.timezone=Canada/Yukon -Druntime.java=24
```

**Applicable branches:**
main

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupState:(initialChildControlState:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:task.keyword,order:0,selectedOptions:!(),title:'GradleTask',type:optionsListControl),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:className.keyword,order:1,selectedOptions:!(org.elasticsearch.multiproject.test.XpackWithMultipleProjectsClientYamlTestSuiteIT),title:'Suite',type:optionsListControl),'144933da-5c1b-4257-a969-7f43455a7901':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:name.keyword,order:2,selectedOptions:!(test%20%7Byaml%3Desql%2F10_basic%2Fbasic%20with%20documents_found%7D),title:'Test',type:optionsListControl)))))

**Failure Message:**
```
java.lang.AssertionError: Failure at [esql/10_basic:200]: 
Expected: <10>
     but: was <8>
```

**Issue Reasons:**
- [main] 2 failures in test test {yaml=esql/10_basic/basic with documents_found} (0.8% fail rate in 245 executions)

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

--- a/muted-tests.yml
+++ b/muted-tests.yml
@@ -417,15 +417,6 @@ tests:
 - class: org.elasticsearch.packaging.test.DockerTests
   method: test026InstallBundledRepositoryPlugins
   issue: https://github.com/elastic/elasticsearch/issues/127081
-- class: org.elasticsearch.multiproject.test.XpackWithMultipleProjectsClientYamlTestSuiteIT
-  method: test {yaml=esql/10_basic/basic with documents_found}
-  issue: https://github.com/elastic/elasticsearch/issues/127088
-- class: org.elasticsearch.xpack.esql.qa.single_node.EsqlClientYamlAsyncIT
-  method: test {p0=esql/10_basic/basic with documents_found}
-  issue: https://github.com/elastic/elasticsearch/issues/127080
-- class: org.elasticsearch.xpack.esql.qa.multi_node.EsqlClientYamlIT
-  method: test {p0=esql/10_basic/basic with documents_found}
-  issue: https://github.com/elastic/elasticsearch/issues/127046
 - class: org.elasticsearch.search.query.QueryPhaseTimeoutTests
   method: testBulkScorerTimeout
   issue: https://github.com/elastic/elasticsearch/issues/127156

